README.md

```diff
@@ -155,13 +155,13 @@ However, at the same time, behavior of modules in general, and the set of module
 
 We see the management of compatibility issues across environments as similar, independent of whether metadata is held in-band or out-of-band. An out of band solution would also suffer from the risk of inconsistent implementation or support across host environments if some kind of coordination does not occur.
 
-The topic of attribute divergence is further discussed in  [#34](https://github.com/tc39/proposal-module-attributes/issues/34).
+The topic of attribute divergence is further discussed in  [#34](https://github.com/tc39/proposal-module-conditions/issues/34).
 
 ### How would this proposal work with caching?
```

Comment:
This section feels outdated, like it's from before we restricted to check attributes. It should be rewritten to express the constraint in the spec (or maybe revised based on the HTML discussion).

Reply:
I think we should also move part of it in the follow-up proposal

Reply:
Sounds good, but let's make sure it's not present in this README with the wrong kind of text. Until we set up that other proposal, it could be edited here along the lines I explained. The text that this PR leaves doesn't make sense.

This FAQ entry should still be present, to explain that import conditions are *not* part of the cache key.

Reply:
I'm unsure what we should write here, apart that conditions are not part of the cache key. The HTML discussion might be specific to the host, and TC39 expressed that we don't wanted it to be part of the cache key already.